app-admin/kubelet-wrapper/files/kubelet-wrapper

```diff
@@ -7,12 +7,36 @@
 
 set -e
 
-if [ -z "${KUBELET_VERSION}" ]; then
-    echo "ERROR: must set KUBELET_VERSION"
+function require_ev_all() {
+    for rev in $@ ; do
+        if [[ -z ${!rev} ]]; then
+            echo ${rev} is not set
+            exit 1
+        fi
+    done
+}
+
+function require_ev_one() {
+    for rev in $@ ; do
+        if [[ ! -z ${!rev} ]]; then
+            return
+        fi
+    done
+    echo One of $@ must be set
     exit 1
-fi
+}
+
+KUBELET_VERSION="${KUBELET_VERSION:-${KUBELET_TAG}}"
+KUBELET_IMG="${KUBELET_ACI}"
 
-KUBELET_ACI="${KUBELET_ACI:-quay.io/coreos/hyperkube}"
+require_ev_one KUBELET_IMG KUBELET_TAG
```

Comment:
@marineam: @aaron, @philips and I went through this discussion too and agreed on _\_TAG_ because it's a commonly understood term for image discovery and a better match than version. The requirement was to make it compatible with _\_VERSION which is fulfilled. I had a bug in the previous version of the PR but it's now updated.